lib/folio/mhld_builder.rb

```diff
@@ -67,7 +67,7 @@ def statments_for_index(holding)
     end
 
     def statements_for_supplements(holding)
-      holding.fetch('holdingsStatementsForSupplements').map { |statement| "Supplement: #{statement.fetch('statement')}" }
+      holding.fetch('holdingsStatementsForSupplements').filter_map { |statement| "Supplement: #{statement.fetch('statement')}" if statement.key?('statement') }
```

Comment:
Could this same thing happen for `holdingsStatementsForIndexes` above?

Reply:
In theory, yes, but we haven't seen any data like that yet.